diff --git a/node_modules/docz/dist/index.esm.js b/node_modules/docz/dist/index.esm.js
index e99902d..ff4a609 100644
--- a/node_modules/docz/dist/index.esm.js
+++ b/node_modules/docz/dist/index.esm.js
@@ -112,7 +112,7 @@ const getTypeStr = (type) => {
             });
             return JSON.stringify(rst, null, 2);
         default:
-            return capitalize(type.name);
+            return type.name;
     }
 };
 const humanize = (type) => getTypeStr(type);
@@ -122,7 +122,7 @@ const getPropType = (prop) => {
     if (!propName)
         return null;
     const isEnum = propName.startsWith('"') || propName === 'enum';
-    const name = capitalize(isEnum ? 'enum' : propName);
+    const name = isEnum ? 'Enum' : propName;
     const value = get('type.value', prop);
     if (!name)
         return null;
