diff --git a/node_modules/docz-theme-default/dist/index.esm.js b/node_modules/docz-theme-default/dist/index.esm.js
index f27e759..9224438 100644
--- a/node_modules/docz-theme-default/dist/index.esm.js
+++ b/node_modules/docz-theme-default/dist/index.esm.js
@@ -27,6 +27,7 @@ import SearchIcon from 'react-feather/dist/icons/search';
 import ChevronDown from 'react-feather/dist/icons/chevron-down';
 import { Location } from '@reach/router';
 import Edit from 'react-feather/dist/icons/edit-2';
+import marked from 'marked';
 
 const white = '#FFFFFF';
 const grayExtraLight = '#F5F6F7';
@@ -1383,7 +1384,7 @@ const PropRequired = styled(PropType) `
 const Props = ({ props, getPropType, }) => {
     const entries = Object.entries(props);
     const components = useComponents();
-    const Paragraph = useMemo(() => styled(components.P || 'p') `
+    const Div = useMemo(() => styled('div') `
       font-size: 16px;
       color: ${get('colors.sidebarTex')};
     `, []);
@@ -1399,7 +1400,7 @@ const Props = ({ props, getPropType, }) => {
                     prop.defaultValue.value.replace(/\'/g, '"'))))),
                 prop.required && (createElement(PropRequired, null,
                     createElement("em", null, "required")))),
-            prop.description && createElement(Paragraph, null, prop.description)));
+            prop.description && createElement(Div, { dangerouslySetInnerHTML: { __html: marked(prop.description) } }, null)));
     })));
 };
 
